lib/nrf_modem_lib/nrf_modem_lib.c

```diff
@@ -130,6 +134,13 @@ static int _nrf_modem_lib_init(const struct device *unused)
 	}
 	k_mutex_unlock(&slist_mutex);
 
+#if defined(CONFIG_NRF_MODEM_LIB_TRACE_LEVEL)
+	err = nrf_modem_lib_trace_start(CONFIG_NRF_MODEM_LIB_TRACE_LEVEL);
```

Comment:
```suggestion
	int err = nrf_modem_lib_trace_start(CONFIG_NRF_MODEM_LIB_TRACE_LEVEL);

```

Reply:
nit: Looks like `err` is only used here. May I suggest declaring it here as well to save a few bytes when `CONFIG_NRF_MODEM_LIB_TRACE_LEVEL` is disabled.

Reply:
Hmm I guess it will be optimized away anyway so no big deal.